dg-override.yaml

```diff
@@ -1,7 +1,7 @@
 name: datagrid/datagrid-8
 version: 1.0
 description: Data Grid Server
-from: ubi8-minimal:8.0-released
+from: ubi8-minimal:ubi-minimal:8.1-407
```

Comment:
Maybe this should be `from: ubi8-minimal:8.1-407`? I would even use `rh-osbs/ubi8-minimal:8.1-407`, because this is the way we reference the images from the Brew registry after the switch to quay.io+proxy.

Reply:
Good spot, I was obviously rushing and I didn't try to build with the override. Updated to `rh-osbs/ubi8-minimal:8.1-407`